server/config.js

```diff
@@ -2,7 +2,13 @@
 
 const path = require('path');
 const log = require('./logger');
-let config = require(path.join(process.cwd(), 'stubborn'));
+let config;
+
+try {
+  config = require(path.join(process.cwd(), 'stubborn'));
+} catch (e) {
+  config = {};
```

Comment:
meh, but if it's optional... 

maybe we postpone the requiring so that we can test if the `options` 👇 exists, if there's no `options` and no `stubborn.js` file then we might `log.error` that